dwave/system/temperatures.py

```diff
@@ -101,11 +101,11 @@ def effective_field(bqm,
        >>> import numpy as np
        >>> from dwave.system.temperatures import effective_field
        >>> N = 5
-       >>> bqm = dimod.BinaryQuadraticModel.from_ising({}, {(i,i+1) : -1 for i in range(N-1)})
+       >>> bqm = dimod.BinaryQuadraticModel.from_ising({}, {(i,i+1) : -0.5 for i in range(N-1)})
        >>> var_labels = list(range(N))
        >>> samples_like = (np.ones(shape=(1,N)), var_labels)
        >>> E = effective_field(bqm,samples_like,current_state_energy=True)
-       >>> print('Cost to flip spin against current assignment', E)
+       >>> print('Cost to flip spin against current assignment', E)     # doctest: +SKIP
```

Comment:
```suggestion
       >>> print('Cost to flip spin against current assignment', E)     
       Cost to flip spin against current assignment [[-1. -2. -2. -2. -1.]]
```

@jackraymond, I tested this locally now and it works fine. I realized this morning that you were getting errors because you were removing the SKIP without adding the expected output. Sorry, I should have caught that yesterday.